ts/Stock/StockToolsBindings.ts

```diff
@@ -499,48 +500,63 @@ extend(NavigationBindings.prototype, {
      * @param {number} defaultHeight
      *        Default height in percents.
      *
+     * @param {boolean} deleteIndicatorAxis
+     *        true, if the indicator is deleted
+     *
      * @return {Array}
      *         An array of calculated positions in percentages.
      *         Format: `{top: Number, height: Number}`
      */
     getYAxisPositions: function (
         yAxes: Array<AxisType>,
         plotHeight: number,
-        defaultHeight: number
+        defaultHeight: number,
+        deleteIndicatorAxis?: boolean
     ): Array<Record<string, number>> {
-        var positions: (Array<Record<string, number>>|undefined),
-            allAxesHeight = 0;
-
+        var positions,
```

Comment:
Can we specify the `positions` type more precisely?
Later in the code, you are casting to `any` a few times.

Reply:
I can specify the type as previously: (Array<Record<string, number>>|undefined)